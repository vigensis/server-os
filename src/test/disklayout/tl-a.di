SCSI	c0t5000CCA02203E181d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c1t5000CCA022040715d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c2t5000CCA022015A91d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c3t5000CCA02203D151d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c4t5000CCA02203E2D1d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c5t5000CCA022016F69d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c6t5000CCA02203D20Dd0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c7t5000CCA022040AB5d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c8t5000CCA022016ADDd0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c9t5000CCA02203E3A9d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c10t5000CCA02203E355d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c11t5000CCA02200C695d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c12t5000CCA0490428ADd0	HGST	HUSMH8010BSS204	100030242816	no	yes
SCSI	c13t5000CCA016011879d0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c14t5000CCA016010FBDd0	HITACHI	HUC109060CSS600	600127266816	no	no
SCSI	c15t5000CCA02203D63Dd0	HITACHI	HUC109060CSS600	600127266816	no	no
USB	c16t0d0	Kingston	DTR30G2	15737028608	yes	no
