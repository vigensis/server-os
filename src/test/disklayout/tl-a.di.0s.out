{
	"vdevs": [
		{
			"type": "raidz",
			"devices": [
				{
					"name": "c0t5000CCA02203E181d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "c10t5000CCA02203E355d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "c11t5000CCA02200C695d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				}
			]
		},
		{
			"type": "raidz",
			"devices": [
				{
					"name": "c13t5000CCA016011879d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "c14t5000CCA016010FBDd0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "c15t5000CCA02203D63Dd0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				}
			]
		},
		{
			"type": "raidz",
			"devices": [
				{
					"name": "c1t5000CCA022040715d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "c2t5000CCA022015A91d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "c3t5000CCA02203D151d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				}
			]
		},
		{
			"type": "raidz",
			"devices": [
				{
					"name": "c4t5000CCA02203E2D1d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "c5t5000CCA022016F69d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "c6t5000CCA02203D20Dd0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				}
			]
		},
		{
			"type": "raidz",
			"devices": [
				{
					"name": "c7t5000CCA022040AB5d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "c8t5000CCA022016ADDd0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				},
				{
					"name": "c9t5000CCA02203E3A9d0",
					"vid": "HITACHI",
					"pid": "HUC109060CSS600",
					"size": "600127266816",
					"solid_state": false
				}
			]
		}
	],
	"capacity": 6001272668160,
	"logs": [
		{
			"name": "c12t5000CCA0490428ADd0",
			"vid": "HGST",
			"pid": "HUSMH8010BSS204",
			"size": "100030242816",
			"solid_state": true
		}
	]
}
